package.json

```diff
@@ -18,7 +18,7 @@
     },
     "scripts": {
         "start": "tsc && unbuffer node build/src/main.js 2>&1 | tee -a log.txt",
-        "lint": "eslint src",
+        "lint": "eslint .",
```

Comment:
This doubles the lint time on my machine. Unless that can be addressed, let's leave it as just src for now. Everything else is a one-time script and I'm not concerned about them.